code/modules/reagents/oldchem/reagents/reagents_misc.dm

```diff
@@ -114,6 +114,11 @@
 	reagent_state = SOLID
 	color = "#D0D0D0" // rgb: 208, 208, 208
 
+/datum/reagent/silver/reaction_mob(mob/living/M, method=TOUCH, reac_volume)
+	if(istype(M) && M.has_bane(BANE_SILVER))
+		M.reagents.add_reagent("toxin", reac_volume)
```

Comment:
same as above for this reagent, and also again, `volume` instead of `reac_volume`